src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -38,15 +38,24 @@ def user_icon(user, size = 20, css_class = nil, alt = nil)
               width: size, height: size, alt: alt, class: css_class)
   end
 
-  def fuzzy_time(time)
+  def fuzzy_time(time, opts = {})
+    output = nil
+
     if Time.now - time < 60
-      return 'now' # rails' 'less than a minute' is a bit long
+      output = 'now' # rails' 'less than a minute' is a bit long
+    else
+      output = time_ago_in_words(time) + ' ago'
     end
-    time_ago_in_words(time) + ' ago'
+
+    if opts["with_fulltime"]
+      output = safe_join(["<span title='#{Time.at(time).to_s}'>".html_safe, "</span>".html_safe], output)
```

Comment:
`Time#at` return something like `2016-10-24 10:04:20 +0200` which I think is not very readable (because of the time zone `+0200` and as it is the local time zone it shouldn't be necessary). I would just render `2016-10-24 10:04` (not sure if we really need the seconds), something like:

`time.strftime("%Y-%m-%d %H:%M")`